src/ripple/app/misc/impl/AmendmentTable.cpp

```diff
@@ -398,9 +398,9 @@ AmendmentTableImpl::AmendmentTableImpl(
             else
             {
                 JLOG(j_.warn())
-                    << "[veto_amendments] section in config has ammendment "
-                    << a.first
-                    << " both [veto_amendments] and [amendments].";
+                    << "[veto_amendments] section in config has amendment "
+                    << '(' << a.first << ", " << a.second
```

Comment:
Shouldn't `a.first` and `a.second` be the same in this case? Assuming that's the case, the previous warning message is better here. The parentheses are unnecessary also.

Reply:
`a.first` is the amendment hash and `a.second` is the amendment name.